src/pyscipopt/scip.pxd

```diff
@@ -656,6 +673,12 @@ cdef extern from "scip/scip.h":
                                      SCIP_BOUNDTYPE* boundtypes,
                                      int* nbranchvars,
                                      int branchvarssize)
+    void SCIPnodeGetAncestorBranchings(SCIP_NODE* node,
+                                     SCIP_VAR** branchvars,
+                                     SCIP_Real* branchbounds,
+                                     SCIP_BOUNDTYPE* boundtypes,
+                                     int* nbranchvars,
+                                     int branchvarssize)
```

Comment:
How is this connected to the functionality of getSparseSols() ?

Reply:
It doesnt. I cleaned it up :)